### PR TITLE
Deprecate snow

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

fixes: https://github.com/demisto/etc/issues/24447

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.yml [hunk 1]
```diff
@@ -65,8 +65,9 @@ configuration:
   name: get_attachments
   required: false
   type: 8
-description: IT service management
+description: Deprecated - We recommend using ServiceNow v2 instead.
 display: ServiceNow
```
Comment (liorblob) [inline Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.yml:69]: Shouldn't we change the display as well? (`ServiceNow - Deprecated`)

**Comment by bakatzir** [comment]
reference: https://github.com/demisto/etc/issues/23291

File: Packs/ServiceNow/Integrations/ServiceNow/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Deprecated - We recommend using ServiceNow v2 instead(available from 5.0.0).
```
Comment (amshamah419) [inline Packs/ServiceNow/Integrations/ServiceNow/CHANGELOG.md:2]:
```suggestion
Deprecated - We recommend using ServiceNow v2 instead (available from 5.0.0).
```

**Comment by amshamah419** [comment]
Just commit the change and you're good to go

**Pull request merged**